.github/workflows/ci-ubuntu.yml

```diff
@@ -6,12 +6,17 @@ jobs:
     strategy:
       matrix:
         perl-version:
-          - '5.30'
+          - "5.30"
     container:
       image: perl:${{matrix.perl-version}}
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2-beta
+      - name: install node 14.x on perl's container
+        run: |
+          apt-get update
+          apt-get install -y curl
+          curl -sL https://deb.nodesource.com/setup_14.x | bash -
+          apt-get install -y nodejs
```

Comment:
I don't understand which value this adds. It also seems like this is not related to the this PR.

Reply:
that would install a recent nodejs version on perl's container.
github actions will fail without nodejs / npm installed